### PR TITLE
handle phantom records

### PR DESCRIPTION
**Pull request by teaforthecat**
If a row is missing from a CVR extract, and that row is part of the random selection, a CVR will be created in it's place along with a discrepancy for every contest in the county. The phantom record CVR will be skipped during the audit, but will appear in the ballot pull list.

### CODE REVIEW

**Comment by teaforthecat** [comment]
This is ready to go. The smoke test passes and manual tests worked too. 
see test/e-1/README.md

**Comment by teaforthecat** [comment]
@gws I noticed there is a conflict with your changes to cvrsUnderAudit in this branch. If this gets merged first, these changes would not be needed and skipNextMaybe could go away.

File: server/eclipse-project/src/main/java/us/freeandfair/corla/asm/ASMTransitionFunction.java [hunk 1]
```diff
@@ -207,10 +207,11 @@ public ASMTransition value() {
     O(new ASMTransition(ROUND_IN_PROGRESS,
                         ROUND_COMPLETE_EVENT,
                         WAITING_FOR_ROUND_SIGN_OFF)),
-    /* We probably want this transition eventually, but not for CDOS
+    /* We probably want this transition eventually, but not for CDOS */
+    // idk, this is what the smoke test is looking for, so let's try it
     EARLY(new ASMTransition(ROUND_IN_PROGRESS,
                             RISK_LIMIT_ACHIEVED_EVENT, 
-                            AUDIT_COMPLETE)), */
+                            AUDIT_COMPLETE)),
```
Comment (gws) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/asm/ASMTransitionFunction.java]: Are we planning to land this in master? This seems like a significant change that would warrant some more testing... I can't point to anything concrete but things like clients continuing to submit ACVRs, possibly with discrepancies...

Comment (teaforthecat) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/asm/ASMTransitionFunction.java]: nope I missed this - thanks!

**Pull request merged**